.github/workflows/playwright.yml

```diff
@@ -109,11 +109,23 @@ jobs:
         with:
           persist-credentials: false
 
+      - name: Detect package manager
+        id: package-manager
+        uses: grafana/plugin-actions/package-manager-detect@package-manager-detect/v1.0.0
+        with:
+          working-directory: ${{ inputs.plugin-directory }}
+
+      - name: Install pnpm
+        if: steps.package-manager.outputs.name == 'pnpm'
+        uses: pnpm/action-setup@a7487c7e89a18df4991f7f222e4898a00d66ddda # v4.1.0
```

Comment:
Nit: We can probably refactor this to avoid repeating it both here and in the setup action (maybe calling the setup action in this step, and add somehing like a `frontend-only` input to the setup action to skip the backend setup steps), but we can leave it for a later follow-up PR